modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/utils/client_extensions/loadEditorClientExtensions.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import loadClientExtensions from './loadClientExtensions';
+
+interface IConfig {
+	editorTransformerURLs: Array<string>;
+}
+
+export default function loadEditorClientExtensions({
```

Comment:
@dsanz We talked about this in https://github.com/liferay-frontend/liferay-portal/pull/3902#discussion_r1435298104. I decided to put the util in `frontend-js-web`, for simplicity and consistency. It's probably not wise to fragment places that fill `Liferay.Util` namespace.

Reply:
nice!

Looking at it, I wonder whether we shall have "spetialized" versions of `load*ClientExtensions()` in the front end, besides the generic one.

The editor CX case deserves this for sure as we have a very specific, reusable semantics across different editor flavors, i.e. multiple injection points for the same CX type (AE, CKEditor), with the additional peculiarity that more than one CX instance may be loaded for the same injection point at runtime.

Normally, the situation is the opposite. We have only one JS-based injection point per CX type (which injects 0 or 1 CX instance each time), so it's general-purpose form `loadClientExtension()` is enough. As soon as we need the same injection logic in separate places, we'll need to build the corresponding abstraction and make it available.